api/scpca_portal/management/commands/load_data.py

```diff
@@ -98,7 +173,7 @@ def clean_up_output_data():
 
     def load_data(
         self,
-        allowed_submitters: set[str] = None,
+        allowed_submitters: set[str] = ALLOWED_SUBMITTERS,
```

Comment:
@davidsmejia now that I'm thinking about it, maybe we should consider just getting rid of this parameter altogether. Is there a use case that we have to support where the `allowed_submitters` might different from the `ALLOWED_SUBMITTERS` constant defined above?